src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,48 +1,67 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+
+type Props = {
+  selectedTodo: Todo | null;
```

Comment:
```suggestion
  selectedTodo: Todo;
```
render modal only if todo selected